lib/test/karma.js

```diff
@@ -38,29 +43,35 @@ function getKarmaReporterPluginPath(reporterName) {
     throw 'Get Karma Reporter Plugin Path: Reporter name is unspecified';
   }
 
-  if (reporterName === defaultReporterName) {
-    return defaultReporterName;
+  //Default reporter is a dependency of angularity
+  var reporterPath = (reporterName === defaultReporterName) ?
+    path.resolve(__dirname, '../../node_modules', defaultReporterName) :
+    reporterName;
+  //Specific reporters are expected to be dependencies of the local project
+  reporterPath = (path.dirname(reporterPath) === '.') ?
+    path.resolve('node_modules', 'karma-' + reporterName + '-reporter') :
```

Comment:
karma's own convention is to specify it without the prefix and suffix.
for example, if you wanted to use karma-junit-reporter, you would specify just 'junit' in your `reporters` array in the config